antarest/study/business/st_storage_manager.py

```diff
@@ -445,29 +445,30 @@ def update_storage(
         values = new_config.dict(by_alias=False)
         return StorageOutput(**values)
 
-    def delete_storage(
+    def delete_storages(
         self,
         study: Study,
         area_id: str,
-        storage_id: str,
+        form: List[str],
     ) -> None:
         """
-        Delete a short-term storage configuration form the given study and area_id.
+        Delete  short-term storages configuration form the given study and area_id.
```

Comment:
The corrected sentence is: "Delete short-term storage configurations from the given study and area."